app/assets/stylesheets/components/_footer.scss

```diff
@@ -26,9 +25,19 @@
   opacity: 1;
 }
 .footer .fa-heart {
-  color: #d23333;
+  color: $light-gray;
 }
 
 .footer-copyright img {
   width: 20px;
 }
+
+@media screen and ( max-width:749px ){
+
+.footer {
+    height: auto;padding: 0;
```

Comment:
careful with two statements in the same line @Leprof20

Reply:
Thanks, fixed now